packages/flutter/lib/src/widgets/color_filter.dart

```diff
@@ -11,7 +11,53 @@ import 'framework.dart';
 
 /// Applies a [ColorFilter] to its child.
 ///
+/// Causes the child to be modified using a color to tint the resulting rendering.
```

Comment:
IMO this is a bit of an understatement. There are other [ColorFilter](https://master-api.flutter.dev/flutter/dart-ui/ColorFilter-class.html) types than the `blend mode` color filter. 

I think it would be useful if you briefly explain what a `ColorFilter` is and maybe add a link to the class in the "seealso" section.

Reply:
Also probably worth mentioning how is this widget different from `BackdropFilter` or `ImageFiltered`.